examples/parallelCoordinates.html

```diff
@@ -12,6 +12,9 @@
         }
         svg {
             display: block;
+            float: left;
+            height: 450px !important;
+            width: 850px !important;
```

Comment:
ah, I see... is there a reason you need two graphs or can you fit all the features you want to demo into one?  If you need 2 that's fine.  I really need to make a way to page through multiple examples anyway...